articles/tutorials/building_2d_games/09_handling_input/index.md

```diff
@@ -0,0 +1,412 @@
+
+| Property                                                                             | Type                                                              | Description                                   |
+|--------------------------------------------------------------------------------------|-------------------------------------------------------------------|-----------------------------------------------|
+| [**A**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.A)                         | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the A button             |
+| [**B**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.B)                         | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the B button             |
+| [**Back**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.Back)                   | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the Back button          |
+| [**BigButton**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.BigButton)         | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the BigButton button     |
+| [**LeftShoulder**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.LeftShoulder)   | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the LeftShoulder button  |
+| [**LeftStick**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.LeftStick)         | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the LeftStick button     |
+| [**RightShoulder**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.RightShoulder) | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the RightShoulder button |
+| [**RightStick**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.RightStick)       | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the RightStick button    |
+| [**Start**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.Start)                 | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the Start button         |
+| [**X**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.X)                         | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the X button             |
+| [**Y**](xref:Microsoft.Xna.Framework.Input.GamePadButtons.Y)                         | [**ButtonState**](xref:Microsoft.Xna.Framework.Input.ButtonState) | Returns the state of the Y button             |
+
+> [!NOTE]
+> Recall from [Chapter 01](../01_what_is_monogame/index.md) that MonoGame is a implementation the XNA API.  Since XNA was originally created for making games on Windows PC and Xbox 360, the names of the gamepad buttons match those of an Xbox 360 controller.

```

Comment:
Good note, but also include an example, for instance a PlayStation controller :D